addons/web/__manifest__.py

```diff
@@ -87,6 +87,7 @@
             'web/static/lib/moment/moment.js',
             'web/static/lib/luxon/luxon.js',
             'web/static/lib/owl/owl.js',
+            'web/static/lib/owl/odoo_module.js',
```

Comment:
Note that `'web/static/lib/owl/*.js'` doesn't work because "odoo_module.js" is loaded before "owl.js".